src/paginate.ts

```diff
@@ -224,6 +225,10 @@ export async function paginate<T>(
         }
     }
 
+    if (config.selectableColumns?.length > 0) {
+        queryBuilder.select(config.selectableColumns)
```

Comment:
how does it behave then passing a query builder which already defined a `.select`?

potentially only allow when passing a repo?

Reply:
it will would replace the defined `.select`. I don't really know how to properly check if the query builder already has it so I don't override it.

Reply:
cool, then let's simply check: if not `Repository` and config `select` is defined, then throw `ServiceUnavailableException`

See https://github.com/ppetzold/nestjs-paginate/blob/master/src/paginate.ts#L204

Reply:
I'm sorry, I am not following 😅. Why would we need to throw that error if `select` is not defined?. Couple of things:
1. I am setting the `select` columns after `queryBuilder` is defined, which is ultimately used to query the DB.
2. `select` is optional. By default, typeorm will return all columns.
Would you mind helping me see what you are seeing? 😬 

Thank You!!

Reply:
Hahah. Nevermind. I thought we may discourage wrong usage. But I suppose it's fine anyways. Should be obvious that it's being overridden.